Server/Test_Plan_External.md

```diff
@@ -0,0 +1,33 @@
+## 'External' Test Plan
+
+- Run Test plan by setting up ownCloud in http and https
+
+### :heavy_exclamation_mark: note
+- Some sites dont allow rendering from different origin. In that case external sites may not load in ownCloud. Make sure the external site under test allows rendering from different origin.
+
+## Testing functionality
+
+Test Case | Expected Result | Result | Related Comment
+------------- | -------------- | ----- | ------
+**CLI commands** |  |   |
+|  |   |
+Enable Activity app via CLI using ```occ app:enable activity``` | The apps gets enabled | :construction:  |
+Disable Activity app via CLI using ```occ app:disable activity``` | The apps gets disabled | :construction:  |
+**WEBUI tests** |  |   |
+|  |   |
+The admin adds a new external site using webUI | The external site icon must be visible in the settings menu | :construction: |
+The admin adds invalid url in the add external sites menu | The admin should see a error message in the webUI | :construction: |
+The admin adds a new external site using webUI, logs out and a user logs in | The external site icon must be visible on settings menu | :construction: |
```

Comment:
The result here should be something that the user sees? Or do I not understand.

Reply:
Yes an icon for external site must appear in the apps menu.

Reply:
The text says "settings menu" but your comment says "apps menu" ?
I better look at what the UI for this app looks like so I know what I am reviewing ;)

Reply:
I actually added settings menu on all steps by mistake but I think it should be app menu. anyways It's the menu on the top left side. What do we call that? since it contains `files`, `Users` and other apps we install later, I think apps menu is appropriate.